docker/ssp-debug-sp/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM webdevops/php-nginx:7.2 AS ssp-debug-sp
+MAINTAINER Michiel Kodde (michiel@ibuildings.nl)
+
+# Install required applications & binaries to install SimpleSAMLphp
+RUN apt-get update && apt-get install -y git python zip libpng-dev nodejs
+RUN docker-php-ext-install pdo_mysql exif gd
+
+# Install Composer
+COPY --from=composer /usr/bin/composer /usr/local/bin/composer
+
+# Fix npm
+RUN mkdir /.npm && chown -R "${NPM_UID}:${NPM_GID}" "/.npm"
+RUN mkdir /.config && chown -R "${NPM_UID}:${NPM_GID}" "/.config"
+
+# Install SSP: Clone and install rev adf1eb8 of SSP
+WORKDIR /var/simplesamlphp/
+RUN git clone https://github.com/simplesamlphp/simplesamlphp.git /var/simplesamlphp
+RUN git reset --hard adf1eb8
```

Comment:
Why the --hard reset here? And not a tag or a version?

Reply:
Good point, I want to install this specific revision as it is also used in the Stepup-deploy SSP task. Using `gir reset` works for me in that respect. 

As a step forward for this, I'd like to go to a more recent SSP version. But we might not be compatible with those versions yet. Given we use some older SSP constructions in the debug sp module.